File: backend/ws-service/src/main/java/com/example/wsservice/fcm/dto/FcmUserMessageDto.java
package com.example.wsservice.fcm.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Map;

/** 방(topic)이 아니라 한 사람에게만 보내는 푸시. 반응 알림은 메시지 작성자 한 명에게만 간다. */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class FcmUserMessageDto {
    private String userId;
    private String title;
    private String body;
    private Map<String, String> data;
}

File: backend/ws-service/src/main/kotlin/com/example/wsservice/fcm/dto/FcmUserMessageDto.kt
package com.example.wsservice.fcm.dto

/** 방(topic)이 아니라 한 사람에게만 보내는 푸시. 반응 알림은 메시지 작성자 한 명에게만 간다. */
data class FcmUserMessageDto(
    var userId: String? = null,
    var title: String? = null,
    var body: String? = null,
    var data: Map<String, String>? = null,
)
